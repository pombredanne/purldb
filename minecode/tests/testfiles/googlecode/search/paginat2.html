


<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOINDEX,NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="https://ssl.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"domainName": null, "token": null, "projectName": null, "relativeBaseUrl": "", "projectHomeUrl": null, "profileUrl": null, "assetVersionPath": "https://ssl.gstatic.com/codesite/ph/11512667522589661435", "assetHostPath": "https://ssl.gstatic.com/codesite/ph", "loggedInUserEmail": null};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 
 
 
 
 Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/11512667522589661435/css/ph_list.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .labels a:link { color: #080 }
 .labels a:visited { color: #080 }
 .labels a:active { color: #f00 }
 .name { margin-top: 2ex; font-size: 120% }

 
 </style>
</head>
<body class="">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=https%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch%3Fq%3Dlabel%253AWebService%26filter%3D0%26mode%3D%26start%3D60&amp;followup=https%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch%3Fq%3Dlabel%253AWebService%26filter%3D0%26mode%3D%26start%3D60" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 >
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="">
 <a href="/">
 
 <img src="https://ssl.gstatic.com/codesite/ph/images/code_small.png" alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 </td>
 <td style="white-space:nowrap;">
 
 <form action="/hosting/search">
 <input size="40" name="q" value="label:WebService" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 

<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 




<script type="text/javascript">
 function starClick(el, project_name) {
 _CS_toggleStar(
 el,
 {'scope': 'projects',
 'user': '_CURRENT_USER',
 'item': project_name
 },
 true,
 '',
 '',
 '',
 'star_count-' + project_name,
 'plural-' + project_name);
 }
</script>
<table class="mainhdr" width="100%" border="0" cellspacing="0" cellpadding="0">
<tr><td width="100%"></td>

 <td style="padding:3px 1em; font-weight:normal; white-space:nowrap;">
 Results 61 - 70 of 963
 </td>

</tr>
</table>
<br/>
<div id="serp">

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/epept/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/epept/logo?cct=1396306800.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 61)"
 href="/p/epept/" style="font-size:medium">epept &nbsp;- &nbsp;Fewer Permutations, More Accurate P-Values</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Mar 09, 2012
 &nbsp;-&nbsp;
 Stars: <span id="star_count-epept">5</span>
 </span><br/>
 
 
 

Motivation 
   In computational biology, permutation tests have become a widely used tool to assess the statistical significance of an event under investigation. However, the common way of comput...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Research">Research</a>, 
 
 <a href="/hosting/search?q=label:SystemsBiology">SystemsBiology</a>, 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/crvcl/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/crvcl/logo?cct=1396306800.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 62)"
 href="/p/crvcl/" style="font-size:medium">crvcl &nbsp;- &nbsp;crVCL PHP Framework</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Dec 22, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-crvcl">2</span>
 </span><br/>
 
 
 
is based on PHP 5.2.1 and Javascript 1.4 (or higher).
It provide classes and function for develop easy and fast web applications/webpages with graphical components.
It give you more comfort in dev...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Framework">Framework</a>, 
 
 <a href="/hosting/search?q=label:Toolkit">Toolkit</a>, 
 
 <a href="/hosting/search?q=label:PHP">PHP</a>, 
 
 <a href="/hosting/search?q=label:Ajax">Ajax</a>, 
 
 <a href="/hosting/search?q=label:MVC">MVC</a>, 
 
 <a href="/hosting/search?q=label:Library">Library</a>, 
 
 <a href="/hosting/search?q=label:REST">REST</a>, 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>, 
 
 <a href="/hosting/search?q=label:SEPA">SEPA</a>, 
 
 <a href="/hosting/search?q=label:RSS-Feed">RSS-Feed</a>, 
 
 <a href="/hosting/search?q=label:Sitemap">Sitemap</a>, 
 
 <a href="/hosting/search?q=label:MySQL">MySQL</a>, 
 
 <a href="/hosting/search?q=label:MySQLLoadbalancing">MySQLLoadbalancing</a>, 
 
 <a href="/hosting/search?q=label:SQLite">SQLite</a>, 
 
 <a href="/hosting/search?q=label:RapidApplicationDevelopment">RapidApplicationDevelopment</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/distributome/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 63)"
 href="/p/distributome/" style="font-size:medium">distributome &nbsp;- &nbsp;Distributome - An Interactive Web-based Resource for Probability Distr…</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Last 7 days
 </span><br/>
 
 
 The  Distributome Project ( is an open-source, open content-development project for exploring, discovering, learning and computational utilization of diverse probability distributions. Probability dis...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Academic">Academic</a>, 
 
 <a href="/hosting/search?q=label:Algorithm">Algorithm</a>, 
 
 <a href="/hosting/search?q=label:Analysis">Analysis</a>, 
 
 <a href="/hosting/search?q=label:Applet">Applet</a>, 
 
 <a href="/hosting/search?q=label:Bioinformatics">Bioinformatics</a>, 
 
 <a href="/hosting/search?q=label:Biology">Biology</a>, 
 
 <a href="/hosting/search?q=label:Java">Java</a>, 
 
 <a href="/hosting/search?q=label:Navigation">Navigation</a>, 
 
 <a href="/hosting/search?q=label:Research">Research</a>, 
 
 <a href="/hosting/search?q=label:Simulator">Simulator</a>, 
 
 <a href="/hosting/search?q=label:Statistics">Statistics</a>, 
 
 <a href="/hosting/search?q=label:Tool">Tool</a>, 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>, 
 
 <a href="/hosting/search?q=label:probability">probability</a>, 
 
 <a href="/hosting/search?q=label:education">education</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/bing-maps-java-sdk/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 64)"
 href="/p/bing-maps-java-sdk/" style="font-size:medium">bing-maps-java-sdk &nbsp;- &nbsp;A Java wrapper for Bing Maps Web Services</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Sep 09, 2012
 &nbsp;-&nbsp;
 Stars: <span id="star_count-bing-maps-java-sdk">7</span>
 </span><br/>
 
 
 Microsoft® Bing Maps Web Services is a set of programmable SOAP services that allow you to match addresses to the map, search for points of interest, integrate maps and imagery, return driving direct...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:bing">bing</a>, 
 
 <a href="/hosting/search?q=label:maps">maps</a>, 
 
 <a href="/hosting/search?q=label:webservice">webservice</a>, 
 
 <a href="/hosting/search?q=label:java">java</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/mobiledatanow/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 65)"
 href="/p/mobiledatanow/" style="font-size:medium">mobiledatanow &nbsp;- &nbsp;Query information from databases and webservices via email, sms or ins…</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Nov 05, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-mobiledatanow">18</span>
 </span><br/>
 
 
 
 MobileDataNow enables someone to query databases and webservices via their mobile phone using email, sms or instant messaging.

Since its simply messaging,  MobileDataNow works with all phones. J...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:sms">sms</a>, 
 
 <a href="/hosting/search?q=label:smpp">smpp</a>, 
 
 <a href="/hosting/search?q=label:mobile">mobile</a>, 
 
 <a href="/hosting/search?q=label:database">database</a>, 
 
 <a href="/hosting/search?q=label:email">email</a>, 
 
 <a href="/hosting/search?q=label:im">im</a>, 
 
 <a href="/hosting/search?q=label:java">java</a>, 
 
 <a href="/hosting/search?q=label:webservice">webservice</a>, 
 
 <a href="/hosting/search?q=label:soap">soap</a>, 
 
 <a href="/hosting/search?q=label:imap">imap</a>, 
 
 <a href="/hosting/search?q=label:jdbc">jdbc</a>, 
 
 <a href="/hosting/search?q=label:sql">sql</a>, 
 
 <a href="/hosting/search?q=label:gsm">gsm</a>, 
 
 <a href="/hosting/search?q=label:query">query</a>, 
 
 <a href="/hosting/search?q=label:openlaszlo">openlaszlo</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/jcm-br/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/jcm-br/logo?cct=1396306800.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 66)"
 href="/p/jcm-br/" style="font-size:medium">jcm-br &nbsp;- &nbsp;Atualização Monetária no Sistema Financeiro do Brasil</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Feb 16, 2012
 &nbsp;-&nbsp;
 Stars: <span id="star_count-jcm-br">6</span>
 </span><br/>
 
 
 Aplicativo multiplataforma com interface gráfica para atualização monetária, calculando e planilhando juros e correção monetária, respeitando a cronologia das reformas do padrão monetário, as...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Brasil">Brasil</a>, 
 
 <a href="/hosting/search?q=label:financeiro">financeiro</a>, 
 
 <a href="/hosting/search?q=label:atualização">atualização</a>, 
 
 <a href="/hosting/search?q=label:monetária">monetária</a>, 
 
 <a href="/hosting/search?q=label:juros">juros</a>, 
 
 <a href="/hosting/search?q=label:correção">correção</a>, 
 
 <a href="/hosting/search?q=label:Java">Java</a>, 
 
 <a href="/hosting/search?q=label:webservice">webservice</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/sourcemod-simple-vip-manager/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 67)"
 href="/p/sourcemod-simple-vip-manager/" style="font-size:medium">sourcemod-simple-vip-manager &nbsp;- &nbsp;sourcemod web-based vip automatic system (key based)</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Feb 19, 2013
 </span><br/>
 
 
 The vip system allows any user to activate a vip slot in the game (like counter-strike) without bothering admin. It has web-interface and requires cron sheduler(web-side option) &amp; basic donators inter...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Module">Module</a>, 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>, 
 
 <a href="/hosting/search?q=label:Manage">Manage</a>, 
 
 <a href="/hosting/search?q=label:game">game</a>, 
 
 <a href="/hosting/search?q=label:Automation">Automation</a>, 
 
 <a href="/hosting/search?q=label:plugin">plugin</a>, 
 
 <a href="/hosting/search?q=label:Mod">Mod</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/rag2agal/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 68)"
 href="/p/rag2agal/" style="font-size:medium">rag2agal &nbsp;- &nbsp;Script da norma RAG à norma AGAL da língua galega</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Earlier this year
 </span><br/>
 
 
 Este script converte um texto em galego da norma RAG à norma AGAL.<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Perl">Perl</a>, 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/wshotelbanque/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 69)"
 href="/p/wshotelbanque/" style="font-size:medium">wshotelbanque &nbsp;- &nbsp;web service mini projet</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: Jul 20, 2013
 </span><br/>
 
 
 mini projet<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>, 
 
 <a href="/hosting/search?q=label:j2ee">j2ee</a>, 
 
 <a href="/hosting/search?q=label:University">University</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/webserver-malware-scanner/">
 <img style="max-width:50px;max-height:55px"
 
 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 70)"
 href="/p/webserver-malware-scanner/" style="font-size:medium">webserver-malware-scanner &nbsp;- &nbsp;We will scan your webserver for the latest malware infections and/or m…</a>
 
 <br/>
 
 <span style="color:#676767">
 Updated: May 12, 2013
 &nbsp;-&nbsp;
 Stars: <span id="star_count-webserver-malware-scanner">1</span>
 </span><br/>
 
 
 
Webserver Malware Scanner is a simple script for detecting and analyzing web-based threats. It currently handles exploitkits, shells, obfuscated JavaScript codes, Executables, Iframes and port scans...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Linux">Linux</a>, 
 
 <a href="/hosting/search?q=label:Bash">Bash</a>, 
 
 <a href="/hosting/search?q=label:Webservice">Webservice</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 </div>
 <div style="clear:both">&nbsp;</div>


<div style="text-align:center; font-size:medium; white-space:nowrap; margin-top:1em;">
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=50"><b>&lsaquo;</b> Prev</a>
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=10">2</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=20">3</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=30">4</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=40">5</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=50">6</a>
 
 
 
	7
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=70">8</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=80">9</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=90">10</a>
 
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=100">11</a>
 
 
 <a href="search?q=label%3AWebService&amp;filter=0&amp;mode=&amp;start=70">Next <b>&rsaquo;</b></a>
</div>
<br/>
<br/>
<div style="background:#F0F7F9; border-top:1px solid #6B90DA; border-bottom:1px solid #6B90DA; text-align:center;">
 <br/>
 <form action="/hosting/search">
 <input type="text" size="30" name="q" value="label:WebService">
 <input type="hidden" name="mode" value="">
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 <br/>
</div>
<script type="text/javascript">
function clk(el, cd) {
 if (document.images) {
 (new Image()).src = "/hosting/search?sa=T&ct=res&cd=" + cd + "&url="
 + escape(el.href) + "&q=label%3AWebService";
 }
 return true;
}
</script>
 
 
 
 <script type="text/javascript" src="https://ssl.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

