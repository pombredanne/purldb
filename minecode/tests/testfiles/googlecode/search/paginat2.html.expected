[
  [
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=10", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=100", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=20", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=30", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=40", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=50", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=70", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=80", 
    "http://code.google.com/hosting/search?q=label%3awebservice&filter=0&mode=&start=90", 
    "http://code.google.com/hosting/search?q=label:academic", 
    "http://code.google.com/hosting/search?q=label:ajax", 
    "http://code.google.com/hosting/search?q=label:algorithm", 
    "http://code.google.com/hosting/search?q=label:analysis", 
    "http://code.google.com/hosting/search?q=label:applet", 
    "http://code.google.com/hosting/search?q=label:atualiza\u00e7\u00e3o", 
    "http://code.google.com/hosting/search?q=label:automation", 
    "http://code.google.com/hosting/search?q=label:bash", 
    "http://code.google.com/hosting/search?q=label:bing", 
    "http://code.google.com/hosting/search?q=label:bioinformatics", 
    "http://code.google.com/hosting/search?q=label:biology", 
    "http://code.google.com/hosting/search?q=label:brasil", 
    "http://code.google.com/hosting/search?q=label:corre\u00e7\u00e3o", 
    "http://code.google.com/hosting/search?q=label:database", 
    "http://code.google.com/hosting/search?q=label:education", 
    "http://code.google.com/hosting/search?q=label:email", 
    "http://code.google.com/hosting/search?q=label:financeiro", 
    "http://code.google.com/hosting/search?q=label:framework", 
    "http://code.google.com/hosting/search?q=label:game", 
    "http://code.google.com/hosting/search?q=label:gsm", 
    "http://code.google.com/hosting/search?q=label:im", 
    "http://code.google.com/hosting/search?q=label:imap", 
    "http://code.google.com/hosting/search?q=label:j2ee", 
    "http://code.google.com/hosting/search?q=label:java", 
    "http://code.google.com/hosting/search?q=label:jdbc", 
    "http://code.google.com/hosting/search?q=label:juros", 
    "http://code.google.com/hosting/search?q=label:library", 
    "http://code.google.com/hosting/search?q=label:linux", 
    "http://code.google.com/hosting/search?q=label:manage", 
    "http://code.google.com/hosting/search?q=label:maps", 
    "http://code.google.com/hosting/search?q=label:mobile", 
    "http://code.google.com/hosting/search?q=label:mod", 
    "http://code.google.com/hosting/search?q=label:module", 
    "http://code.google.com/hosting/search?q=label:monet\u00e1ria", 
    "http://code.google.com/hosting/search?q=label:mvc", 
    "http://code.google.com/hosting/search?q=label:mysql", 
    "http://code.google.com/hosting/search?q=label:mysqlloadbalancing", 
    "http://code.google.com/hosting/search?q=label:navigation", 
    "http://code.google.com/hosting/search?q=label:openlaszlo", 
    "http://code.google.com/hosting/search?q=label:perl", 
    "http://code.google.com/hosting/search?q=label:php", 
    "http://code.google.com/hosting/search?q=label:plugin", 
    "http://code.google.com/hosting/search?q=label:probability", 
    "http://code.google.com/hosting/search?q=label:query", 
    "http://code.google.com/hosting/search?q=label:rapidapplicationdevelopment", 
    "http://code.google.com/hosting/search?q=label:research", 
    "http://code.google.com/hosting/search?q=label:rest", 
    "http://code.google.com/hosting/search?q=label:rss-feed", 
    "http://code.google.com/hosting/search?q=label:sepa", 
    "http://code.google.com/hosting/search?q=label:simulator", 
    "http://code.google.com/hosting/search?q=label:sitemap", 
    "http://code.google.com/hosting/search?q=label:smpp", 
    "http://code.google.com/hosting/search?q=label:sms", 
    "http://code.google.com/hosting/search?q=label:soap", 
    "http://code.google.com/hosting/search?q=label:sql", 
    "http://code.google.com/hosting/search?q=label:sqlite", 
    "http://code.google.com/hosting/search?q=label:statistics", 
    "http://code.google.com/hosting/search?q=label:systemsbiology", 
    "http://code.google.com/hosting/search?q=label:tool", 
    "http://code.google.com/hosting/search?q=label:toolkit", 
    "http://code.google.com/hosting/search?q=label:university", 
    "http://code.google.com/hosting/search?q=label:webservice"
  ], 
  [
    {
      "url": "http://code.google.com/p/epept/", 
      "last_updated": "2012-03-09T00:00:00+00:00", 
      "stars": 5
    }, 
    {
      "url": "http://code.google.com/p/crvcl/", 
      "last_updated": "2013-12-22T00:00:00+00:00", 
      "stars": 2
    }, 
    {
      "url": "http://code.google.com/p/distributome/", 
      "last_updated": "2014-01-26T00:00:00+00:00", 
      "stars": 0
    }, 
    {
      "url": "http://code.google.com/p/bing-maps-java-sdk/", 
      "last_updated": "2012-09-09T00:00:00+00:00", 
      "stars": 7
    }, 
    {
      "url": "http://code.google.com/p/mobiledatanow/", 
      "last_updated": "2013-11-05T00:00:00+00:00", 
      "stars": 18
    }, 
    {
      "url": "http://code.google.com/p/jcm-br/", 
      "last_updated": "2012-02-16T00:00:00+00:00", 
      "stars": 6
    }, 
    {
      "url": "http://code.google.com/p/sourcemod-simple-vip-manager/", 
      "last_updated": "2013-02-19T00:00:00+00:00", 
      "stars": 0
    }, 
    {
      "url": "http://code.google.com/p/rag2agal/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 0
    }, 
    {
      "url": "http://code.google.com/p/wshotelbanque/", 
      "last_updated": "2013-07-20T00:00:00+00:00", 
      "stars": 0
    }, 
    {
      "url": "http://code.google.com/p/webserver-malware-scanner/", 
      "last_updated": "2013-05-12T00:00:00+00:00", 
      "stars": 1
    }
  ]
]